src/macros.rs

```diff
@@ -30,23 +31,38 @@
 ///
 /// let p = point!("measure", ("f1", "f1v"));
 /// ```
+///
+/// Creates a point with no tags, one field and a custom timestamp:
+///
+/// ```
+/// use telegraf::point;
+///
+/// let p = point!("measure", ("f1", "f1v"); 100);
+/// ```
 #[macro_export]
 macro_rules! point {
-    ($measure:expr, $(($fname:expr, $fval:expr)) +) => {
+    ($measure:expr, $(($fname:expr, $fval:expr)) + $(; $ts:expr)?) => {
         {
             let mut fields: Vec<(String, Box<dyn $crate::IntoFieldData>)> = Vec::new();
             $(
                 fields.push((String::from($fname), Box::new($fval)));
             )*
 
+            #[allow(unused_mut)]
+            let mut timestamp: Option<u64> = None;
+            $(
+                timestamp = timestamp.or(Some($ts));
+            )?
+
```

Comment:
I think this could be done differently, but honestly I'm so rusty at macros that I cant remember how. We should be able to just use `$ts` but the macro thinks its repeating, which it should not be (there should be only 1 timestamp).